gulpfile.js

```diff
@@ -601,6 +605,52 @@ function printConfigHelp(command) {
   }
 }
 
+/**
+ * Prints a helpful message that lets the developer know how to build
+ * a list of extensions or without any extensions.
+ */
+function printExtensionsHelp() {
```

Comment:
This function does more than just print help messages. It also parses the extension flags. Mabye rename it to `parseExtensionFlags()`?